lib/rspec/matchers/built_in.rb

```diff
@@ -12,6 +12,7 @@ module Matchers
     module BuiltIn
       autoload :BeAKindOf,               'rspec/matchers/built_in/be_kind_of'
       autoload :BeAnInstanceOf,          'rspec/matchers/built_in/be_instance_of'
+      autoload :BeASubclassOf,           'rspec/matchers/built_in/be_subclass_of'
```

Comment:
It seems odd for the primary method to be `be_a_subclass_of` but the implementation file to be `be_subclass_of`.  Can we make these consistent?